src/PrestaShopBundle/Form/Admin/Sell/Order/CancelProductType.php

```diff
@@ -94,8 +121,37 @@ public function buildForm(FormBuilderInterface $builder, array $options): void
                     ],
                 ]
             )
+            ->add('voucher_refund_type', ChoiceType::class,
+                [
+                    'required' => true,
+                    'multiple' => false,
+                    'expanded' => true,
+                    'choices' => [
+                        $this->trans('Product(s) price: ', 'Admin.Orderscustomers.Feature') => VoucherRefundType::PRODUCT_PRICES_REFUND,
+                        $this->trans('Product(s) price, excluding amount of initial voucher: ', 'Admin.Orderscustomers.Feature') => VoucherRefundType::PRODUCT_PRICES_EXCLUDING_VOUCHER_REFUND,
```

Comment:
_.Feature_ or _.Help_, is it a tooltip?
🤔

Reply:
No it's a radio button selection
![Capture d’écran 2020-01-24 à 11 24 35](https://user-images.githubusercontent.com/13801017/73062205-23859a00-3e9c-11ea-8357-5fa63e5fceea.png)

I also need the extra space to concatenate the price after, but I can remove it from the wording and add it programmatically It's just that this wording was already used in legacy

Reply:
Good for me, thanks Jo!